modules/tinymce/src/themes/silver/main/ts/ui/context/ContextUi.ts

```diff
@@ -50,8 +50,8 @@ const renderContextToolbar = (spec: { onEscape: () => Option<boolean>; sink: All
 
     inlineBehaviours: Behaviour.derive([
       AddEventsBehaviour.config('context-toolbar-events', [
-        AlloyEvents.runOnSource<EventArgs>(NativeEvents.transitionend(), (comp, se) => {
-          InlineView.getContent(comp).each((c) => {
+        AlloyEvents.runOnSource<EventArgs>(NativeEvents.transitionend(), (comp, _se) => {
+          InlineView.getContent(comp).each((_c) => {
```

Comment:
This whole function (3 lines?) does nothing, so should we remove it?

Reply:
Yep, will remove.

Reply:
Also the reason for the commented out line is sadly lost in the great squash...

Reply:
Done